api-docs/src/main/kotlin/org/bitcoindevkit/bdk.kt

```diff
@@ -0,0 +1,453 @@
+package org.bitcoindevkit
+
+/**
+ * The cryptocurrency to act on.
+ */
+enum class Network {
+    /** Bitcoin's mainnet */
```

Comment:
Just a consistency nit, but I think all javadocs lines should end with a period.

Reply:
Ok cool I didn't know! I'll make sure they all do.